include/kokkos-utils/view/extents.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef KOKKOS_UTILS_VIEW_EXTENTS_HPP
+#define KOKKOS_UTILS_VIEW_EXTENTS_HPP
+
+#include "kokkos-utils/concepts/View.hpp"
+
+/**
+ * @file
+ *
+ * This file is a collection of helper functions similar to what can be found in
+ * the standard @c mdspan header, related to retrieving the extents of a @c Kokkos::View.
+ */
+
+namespace Kokkos::utils::view
+{
+
+namespace impl
+{
+
+//! Implementation of @ref Kokkos::utils::view::extents.
+template <concepts::View ViewType, size_t... Ints>
+auto extents(const ViewType& view, std::index_sequence<Ints...>)
```

Comment:
C(s)ould it be `KOKKOS_INLINE_FUNCTION constexpr`?

Reply:
Indeed, let's make them all `KOKKOS_INLINE_FUNCTION constexpr`. Done :heavy_check_mark: